functions/server/fn_storeValue.sqf

```diff
@@ -1,12 +1,30 @@
-params ["_array"];
-// _unit,_side,_pos,_dir,_kindof,_veh,_hide
+#include "script_component.hpp"
 
-// prevent markers to render double on the same position (e.g. full team in vehicle)
-/*
-if (!(_vehicle in GRAD_REPLAY_DATABASE_TEMP)) then {
-	GRAD_REPLAY_DATABASE_TEMP = GRAD_REPLAY_DATABASE_TEMP + [_array];
+params [["_currentUnitData",[]],["_nextTickData",[]],["_unitID",-1],["_newUnitData",[]]];
+
+// _newUnitData contents: [_icon,_colorID,_pos,_dir,_name,_groupname]
+if (count _newUnitData == 0) exitWith {};
```

Comment:
`_newUnitData isEqualTo []` is a faster "is empty" check although I don't really like it.

Reply:
Can't confirm on my end. Seem to be the same speed regardless of array size.

Reply:
https://community.bistudio.com/wiki/Code_Optimisation#isEqualTo_and_count